src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -385,6 +389,9 @@ private static String suppressVariationsInInjectedValues(String content) {
     private static String replaceUnpredictableValuesWithPlaceholders(String content) {
         Date now = new Date();
         SimpleDateFormat sdf = new SimpleDateFormat("EEE, dd MMM yyyy, ");
+        // get session's time zone from content (content here includes pages only with one time zone).
```

Comment:
`content here includes pages only with one time zone` is not accurate. Remove this part?